### DISCUSSION TITLE
CaptureStdout  not working with spd log library

### DISCUSSION THREAD

**Discussion by suri-log** [google/googletest#4377] [Community Help]
hi team ,
CaptureStderr,CaptureStdout is not capturing the  info when i use the spdlog (https://github.com/gabime/spdlog.git) library . 
i am using the logger library to log the meassage with different loglevels , but the logs messsagaes are getting logged on stdout but the capture functions are not working if i use the above library ,is there any limitation or open bug on this . 
or can you please let me know how can i use the CaptureStdout , if i try to log messages on stdout with spdlog library stdout sink.

**Comment by jwdinius** [comment]
+1.  I am having the same problem.
